src/CMakeLists.txt

```diff
@@ -49,6 +49,11 @@ target_link_libraries(${PROJECT_LIBRARY_TARGET_NAME}
   PRIVATE
     TINYXML2::TINYXML2)
 
+if (WIN32)
+  target_link_libraries(${PROJECT_LIBRARY_TARGET_NAME}
```

Comment:
Do we want to link against this library only if `GZ_ENABLE_RELOCATABLE_INSTALL` is enabled?

Reply:
I believe `InstallationDirectories.cc` is compiled unconditionally, so I think we can't avoid linking it.

an alternative suggestion from @iche033 in https://github.com/gazebosim/sdformat/pull/1414#issuecomment-2297148801 is to avoid using `shlwapi` by copying the `joinPaths` implementation from `gz-common` and use `std::filesystem`

Reply:
Yeah, that sounds like a good idea.

Reply:
updated implementation in #1469